### PR TITLE
Update Query Creator Readme and samconfig.toml with deployment info

### PR DESCRIPTION
**Pull request by sfentress**

### CODE REVIEW

File: query-creator/README.md [hunk 1]
```diff
@@ -12,14 +12,47 @@ To use the SAM CLI, you need the following tools.
...
+Deploying currently involves three steps:
+
+1. Switching to the appropriate role in the CLI
+2. Running the deploy script
+3. Copying in secrets from the CloudFormation stack
+
+### Deploying staging
+
+The following profile needs to be added to the ~/.aws/config file that is created by the CLI, to allow you to switch
+to the AdminConcordQA role. It assumes that there is already a profile named `default` with admin privilages in the
```
Comment (mklewandowski) [inline query-creator/README.md:26]: "privileges"

File: query-creator/README.md [hunk 2]
```diff
@@ -12,14 +12,47 @@ To use the SAM CLI, you need the following tools.
...
+```
+
+1. Switch to the QA role: `export AWS_PROFILE="concord-qa"`
+2. Run the deploy script with the staging properties: `sam deploy --guided --config-env=staging`
+3. Accept all the defaults (just hit enter) except
+  1. For any non-filled-in secrets (e.g. the ReportServiceToken) find the report-service-query-creator CloudFormation
+     stack under the QA account, switch to the Parameters tab, and copy and paste the values
+  2. For "CreateQueryFunction may not have authorization defined, Is this okay?" answer `y`
+4. If there aren't unexpected changes staged, answer `y` to "Deploy this changeset?"
+5. If new secrets were written to the samconfig.toml file, edit them out before commiting changes
```
Comment (mklewandowski) [inline query-creator/README.md:43]: "committing"

**Review by mklewandowski — APPROVED** [review]
👍 I added comments for a couple of very minor fixes, but otherwise looks good.

File: query-creator/README.md [hunk 3]
```diff
@@ -12,14 +12,47 @@ To use the SAM CLI, you need the following tools.
...
+1. Switch to your default role: `export AWS_PROFILE="default"`
+2. Run the deploy script with the production properties: `sam deploy --guided --config-env=production`
+
+Follow the rest of the steps from step 3 of "Deploying staging", using the CloudFormation stack from the
+production account.
+
+### Deployment properties
 
-The first command will build the source of your application. The second command will package and deploy your application to AWS, with a series of prompts:
+These should all be defind in the samconfig.toml, but here is what they mean:
```
Comment (kswenson) [inline query-creator/README.md:55]:
```suggestion
These should all be defined in the samconfig.toml, but here is what they mean:
```

**Pull request merged**